src/devyatovskaya/Task2/FPSCounter.h

```diff
@@ -0,0 +1,24 @@
+#pragma once
+#include <QObject>
+#include <chrono>
+
+class FPSCounter final : public QObject {
+
+  Q_OBJECT
+signals:
+  void emit_fps(const QString &);
+
+public:
+  void calculate_fps();
+
+  float delta_time() const;
```

Comment:
```suggestion
  [[nodisacrd]] float delta_time() const noexcept;
```
Есть интересный атрибут `[[nodisacrd]]` почитайте. Функции, которые не бросают исключений нужно метить как `noexcept`